sw/otbn/crypto/p256_key_from_seed_sca.s

```diff
@@ -53,33 +53,35 @@ run_gen_secret_key:
 
   /* Load shares of seed from DMEM.
        [w21,w20] <= dmem[seed0]
-       [w23,w33] <= dmem[seed1] */
+       [w11,w10] <= dmem[seed1] */
   li        x2, 20
   la        x3, seed0
   bn.lid    x2, 0(x3++)
   li        x2, 21
   bn.lid    x2++, 0(x3)
```

Comment:
nit: The `++` here is now unnecessary; same for the similar block below.

Reply:
Thanks! Removed it.